src/app.ts

```diff
@@ -0,0 +1,27 @@
+import router from './config/routing';
+import { RestService } from './services/rest.service';
+import { CardComponent } from './components/card/card.component';
+
+import './styles/main.scss';
+
+function start() {
+  window['appContext'] = (function() {
+    function appContext() {
```

Comment:
I think would be better using an ES6 Class for this and also have it in a separate file